include/orc/dwarf_structs.hpp

```diff
@@ -120,7 +109,6 @@ struct attribute_value {
     std::uint64_t _uint{0};
     std::int64_t _int{0};
     pool_string _string;
-    const struct die* _die{nullptr};
```

Comment:
I was able to remove this field, which is a pointer per attribute, so that's a win. Probably more space saved than we eat by adding `_cu_die_address`.